scripts/update_python.py

```diff
@@ -32,27 +33,35 @@
 from subprocess import check_call
 from zip import unzip_cmd, zip_cmd
 
-version = '3.9.2'
+version = '3.13.3'
 major_minor_version = '.'.join(version.split('.')[:2])  # e.g. '3.9.2' -> '3.9'
-download_url = 'https://www.nuget.org/api/v2/package/python/%s' % version
 # This is not part of official Python version, but a repackaging number appended by emsdk
 # when a version of Python needs to be redownloaded.
-revision = '4'
+revision = '0'
 
-pywin32_version = '227'
-pywin32_base = 'https://github.com/mhammond/pywin32/releases/download/b%s/' % pywin32_version
+PSUTIL = 'psutil==7.0.0'
 
 upload_base = 'gs://webassembly/emscripten-releases-builds/deps/'
 
 
+# Detects whether current python interpreter architecture is ARM64 or AMD64
+# If running AMD64 python on an ARM64 Windows, this still intentionally returns AMD64
```

Comment:
I'm not sure I understand the second part of this comment.

Do we really want to report `AMD64` if we are running amd64 python on arm64 windows?

Reply:
I see, so effectively this script cannot cross build the arm64 python package.